Source/WebCore/style/ColorFromPrimitiveValue.cpp

```diff
@@ -65,14 +65,14 @@ StyleColor colorFromPrimitiveValue(const Document& document, RenderStyle& style,
 Color colorFromPrimitiveValueWithResolvedCurrentColor(const Document& document, RenderStyle& style, const CSSPrimitiveValue& value)
 {
     // FIXME: 'currentcolor' should be resolved at use time to make it inherit correctly. https://bugs.webkit.org/show_bug.cgi?id=210005
-    if (StyleColor::isCurrentColor(value)) {
+    if (StyleColor::containsCurrentColor(value)) {
```

Comment:
I've caused a Speedometer regression by using `StyleColor::containsCurrentColor` (the version that takes a CSSPrimitiveValue) so might be worth A / B testing. If it does cause a speedometer regression, it would be cool to look into optimizing this method.

Reply:
Just letting you know `StyleColor::containsCurrentColor` caused a regression, given shadows are used by Speedometer, I'd still A / B test to avoid the patch getting reverted later on.

Reply:
A/B testing ✅ Speedometer doesn't show any regression with this patch 👍